src/links/links.macros.html

```diff
@@ -8,18 +8,29 @@
 {% from '../icons/icons.macros.html' import icon as icon_macro %}
 
 {#
+  * Renders a link
+  * @param {string} content - Link text content
+  * @param {string} href - Path or full URL for link
+  * @param {string} [class] - Class name for style or JS targeting
+  * @param {string} [id] - ID for JS targeting
+#}
+{% macro link(content, href, class=null, id=null) %}
+<a class="link{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %} href="{{ href }}">
+  {{ content }}
+</a>
+{% endmacro %}
```

Comment:
I'm ok with not having a link macro, would all link styling then be a part of typography?